### DISCUSSION TITLE
SOLVED - Add an Insert not working

### DISCUSSION THREAD

**Discussion by sagetone** [apohl79/audiogridder#884] [Help] [answered]
Hi, in 1.2.0 beta 6, the Add an insert function is not working.  Left click, right click, double click on the '+' is not doing anything.  Is this a bug, or a user error?  :)

![image](https://user-images.githubusercontent.com/17507376/148661303-289221ba-975d-4c29-b0d6-af72598028ff.png)

**Comment by sagetone** [comment, marked as answer]
It was user error.  :)  When I add an instance of AudioGridder and **_not_** AudioGridderInst, it works just fine.  (DAW is Bitwig).

![image](https://user-images.githubusercontent.com/17507376/148662491-eed31bf7-8070-4a06-bd4d-80a2edec51db.png)

**Comment by apohl79** [comment]
You can only load a single instrument. You need to add an AG FX plugin to add effects plugins behind your instrument output.

**Reply by sagetone** [reply]
Thank you Andreas. I have just started using AG this weekend. Working with it some more today, I am finding that the AG plugs can find all of my instruments when I use AudioGridderInst but not AudioGridder, and of course I have to use AudioGridder to find all of my FX.

I'm still learning it. But really loving your work thus far!
